dlog/kimchi/src/index.rs

```diff
@@ -144,6 +144,55 @@ pub struct VerifierIndex<G: CommitmentCurve> {
     pub fq_sponge_params: ArithmeticSpongeParams<Fq<G>>,
 }
 
+pub fn expr_linearization<F: FftField + SquareRootField>(
+    domain: D<F>,
+    lookup_used: bool,
+    chacha: bool,
+    dummy_lookup_value: Option<&[F]>,
```

Comment:
Nit: use `dummy_loolup_value` as the signaller for `lookup_used`.

Reply:
signaller?

Reply:
Infer `lookuo_used = true` when it's `Some`, `false` when it's `None`.